src/subscription/UpgradeConfirmSubscriptionPage.ts

```diff
@@ -33,7 +32,7 @@ export class UpgradeConfirmPage implements WizardPageN<UpgradeSubscriptionData>
 	}
 
 	view({ attrs }: Vnode<WizardPageAttrs<UpgradeSubscriptionData>>): Children {
-		return [attrs.data.type === SubscriptionType.Free ? this.renderFree(attrs) : this.renderPaid(attrs)]
+		return attrs.data.type === SubscriptionType.Free ? [] : this.renderPaid(attrs) // should never be free
```

Comment:
I feel like we can just inline the renderPaid method then

Reply:
yea that's true